docs/build/guides/basics/automate-reset-data.mdx

```diff
@@ -0,0 +1,460 @@
+      )
+      .setTimeout(30)
+      .build();
+
+    const invokeContractTx = await server.prepareTransaction(
+      invokeContractTransaction,
+    );
+    invokeContractTx.sign(deployer);
+    let invokeContractResponse = await server.sendTransaction(invokeContractTx);
+    const returnInvokeContractResponse = await getTxsReports(
+      invokeContractResponse,
+      `Invoke Contract.`,
+      server,
+    );
+
+    const functionValues = returnInvokeContractResponse.returnValue._value
+      .map((item) => item._value)
+      .filter(Boolean);
+
+    return {
+      contractId: contractId,
+      returnValues: functionValues,
+    };
+  } catch (error) {
+    console.error("Error in contract deployment and invocation:", error);
+    throw error;
+  }
+}
+
+async function automateSetup() {
+  try {
+    // const networkRPC = "USE EITHER FUTERNET OR TESTNET RPC"
+    const networkRPC = "https://soroban-testnet.stellar.org";
+    // Example
+    // FOR FUTERENET - https://rpc-futurenet.stellar.org
+    // FOR TESTNET - https://soroban-testnet.stellar.org
+
+    //Initiate Server
+
+    const server = new StellarSdk.SorobanRpc.Server(networkRPC);
+
+    //Check Network Status
+    console.log("Checking network health...");
+    const health = await server.getHealth();
+    console.log("Network health:", health);
+
+    // const networkURL = "USE EITHER FUTERNET OR TESTNET URL"
+    const networkURL = "https://friendbot.stellar.org";
+    // Example
+    // FOR FUTERENET - https://friendbot-futurenet.stellar.org
+    // FOR TESTNET - https://friendbot.stellar.org
+
+    const accountOne = await createAccount(networkURL);
+    const accountTwo = await createAccount(networkURL);
+
+    // const networkParaphrase = "USE EITHER FUTERNET OR TESTNET Paraphrase"
+    const networkPassPhrase = StellarSdk.Networks.TESTNET;
+    // Example
+    // FOR FUTERENET - StellarSdk.Networks.FUTURENET
+    // FOR TESTNET -  StellarSdk.Networks.TESTNET
+
+    console.log("Issuing an Asset...");
+    // Issue assets to these accounts
+    const customAsset = new StellarSdk.Asset("Boya", accountOne.publicKey());
+    await issueAsset(
+      server,
+      networkPassPhrase,
+      accountOne,
+      accountTwo,
+      customAsset,
+    );
+
+    // Create liquidity pool
+    console.log("Creating liquidity pool...");
+    const nativeAsset = StellarSdk.Asset.native();
+    await createLiquidityPool(
+      server,
+      accountOne,
+      nativeAsset,
+      customAsset,
+      networkPassPhrase,
+    );
+
+    // Deploy a contract
+    console.log("Deploying contract...");
+    // Ensure you have the contract Wasm file compiled and saved in the specified path.
+    // Adjust this path as necessary
+    const contractWasmFilePath =
+      "./target/wasm32-unknown-unknown/release/hello_world.wasm"; // This is Wasmfile for an Hello World Contract.
+    const salt =
+      "3157dca02e4105a749bd88d0ea429cde67b399aaa99dc646edccd74d5692a857"; // example salt
```

Comment:
Feel free to drop in your own "easter egg" into the `salt` here. For example `KodeSage was here!` in hex format would be `4b6f64655361676520776173206865726521`. Obviously, you don't have to, but I'm a dork and I like to do stuff like that 🤣